WordPress/Classes/ViewRelated/People/PeopleViewController.swift

```diff
@@ -335,14 +342,30 @@ open class PeopleViewController: UITableViewController, NSFetchedResultsControll
             return
         }
 
-        noResultsView.titleText = NSLocalizedString("No \(filter.title) Yet",
-            comment: "Empty state message (People Management). Please, do not translate the \\(filter.title) part!")
+        noResultsView.titleText = noResultsTitle()
 
         if noResultsView.superview == nil {
             tableView.addSubview(withFadeAnimation: noResultsView)
         }
     }
 
+    private func noResultsTitle() -> String {
+        let noPeople = NSLocalizedString("No \(filter.title) Yet",
```

Comment:
For translation reasons we can't use this format when composing strings, we have to always do %@, sorry 😬